tests/callbacks/test_tqdm_progress_bar.py

```diff
@@ -276,36 +354,16 @@ def test_tqdm_progress_bar_value_on_colab(tmpdir):
     assert trainer.progress_bar_callback.refresh_rate == 19
 
 
-class MockTqdm(Tqdm):
-    def __init__(self, *args, **kwargs):
-        self.n_values = []
-        super().__init__(*args, **kwargs)
-        self.__n = 0
-        # again to reset additions from `super().__init__`
-        self.n_values = []
-
-    @property
-    def n(self):
-        return self.__n
-
-    @n.setter
-    def n(self, value):
-        self.__n = value
-        # track the changes in the `n` value
-        if not len(self.n_values) or value != self.n_values[-1]:
-            self.n_values.append(value)
-
-
 @pytest.mark.parametrize(
     "train_batches,val_batches,refresh_rate,train_updates,val_updates",
     [
         [2, 3, 1, [1, 2, 3, 4, 5], [1, 2, 3]],
         [0, 0, 3, None, None],
         [1, 0, 3, [1], None],
-        [1, 1, 3, [2], [1]],
+        [1, 1, 3, [1, 2], [1]],
         [5, 0, 3, [3, 5], None],
-        [5, 2, 3, [3, 7], [2]],
-        [5, 2, 6, [7], [2]],
+        [5, 2, 3, [3, 5, 7], [2]],
```

Comment:
I don't see how these numbers are changing again #11213
we're not using multiple loaders here

Reply:
yes, we are not but the way the progress is being tracked now is changed to support multiple-dataloaders.
here:
refresh_rate is 3, total train batches = 5
now the first update happens at 3, the second one should happen at 6, but since train batches are limited at 5 (2 more batches), we need to update at 5 too and start with validation after that.